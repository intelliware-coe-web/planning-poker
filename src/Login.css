.Login {
    text-align: center;
}
  
.Login-header {
    background-color: #282c34;
    min-height: 100vh;
    display: flex;
    flex-direction: column;
    align-items: center;
    justify-content: center;
    font-size: calc(10px + 2vmin);
    color: white;
}

.Login-link {
    color: #61dafb;
}

.Login-form-container {
    width: 100%;
    margin-top: 15%;
}

.Login-form {
    width: 400px;
}

.uk-inline {
    display: block;
}