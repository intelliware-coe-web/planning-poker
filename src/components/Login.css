.login {
    text-align: center;
}

.login-form-container {
    width: 100%;
    margin-top: 10%;
}

.login-form {
    width: 400px;
}

.uk-inline {
    display: block;
}