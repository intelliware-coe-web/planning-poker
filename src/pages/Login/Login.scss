.login-logo {
  height: 20vmin;
  margin-bottom: 20px;
}

.login {
  display: flex;
  //flex-direction: column;
  align-items: center;
  min-height: 100vh;
  justify-content: center;
}

@media screen and (max-width: 1024px) {
  .login {
    flex-direction: column;
  }
}