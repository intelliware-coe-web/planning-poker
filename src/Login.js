import React from 'react';
import './Login.css';

function Login() {
  return (
    <div className="Login">
      <div className="Login-form-container">
        <div className="uk-align-center Login-form">
        <span uk-icon="icon: users; ratio: 5" className="uk-margin"></span>
        <form>
          <fieldset className="uk-fieldset">
            <div className="uk-margin">
              <div className="uk-inline">
                <span className="uk-form-icon" uk-icon="icon: user"></span><input className="uk-input" type="text" placeholder="Username"></input>
              </div>
            </div>
            <div className="uk-margin">
              <div className="uk-inline">    
                <span className="uk-form-icon uk-form-icon-flip" uk-icon="icon: lock"></span>
                <input className="uk-input" type="text" placeholder="Password"></input>
              </div>
            </div>
          </fieldset>
          <div className="uk-inline">
            <button className="uk-button uk-button-primary uk-width-1-1 uk-margin-small-bottom">Login</button>
          </div>
        </form>
        </div>
      </div>
     </div>
  );
}

export default Login;