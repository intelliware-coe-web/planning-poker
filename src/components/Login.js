import React from 'react';
import { Link } from "react-router-dom";
import './Login.css';

function Login() {
  return (
    <div className="login">
      <div className="login-form-container">
        <div className="uk-align-center login-form">
        <span uk-icon="icon: users; ratio: 5" className="uk-margin"></span>
        <form>
          <fieldset className="uk-fieldset">
            <div className="uk-margin">
              <div className="uk-inline">
                <span className="uk-form-icon" uk-icon="icon: user"></span><input className="uk-input" type="text" placeholder="Username"></input>
              </div>
            </div>
            <div className="uk-margin">
              <div className="uk-inline">    
                <span className="uk-form-icon uk-form-icon-flip" uk-icon="icon: lock"></span>
                <input className="uk-input" type="text" placeholder="Password"></input>
              </div>
            </div>
          </fieldset>
          <div className="uk-inline">
            <Link to="/room">
              <button className="uk-button uk-button-primary uk-width-1-1 uk-margin-small-bottom">
                Login
              </button>
            </Link>
          </div>
        </form>
        </div>
      </div>
     </div>
  );
}

export default Login;