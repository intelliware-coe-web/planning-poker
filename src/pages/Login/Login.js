import React from 'react';
import EmailInput from "../../components/EmailInput";
import PasswordInput from "../../components/PasswordInput";
import Button from "../../components/Button";
import logo from '../../logo.svg';
import './Login.scss';
import {Link} from "react-router-dom";

function Login() {
    return (
        <div className="login">
            <p>Welcome to Planning Poker</p>
            <img src={logo} className="login-logo" alt="logo" />
            <form>
                <EmailInput/>
                <PasswordInput/>
                <Link to="/rooms">
                    <Button name={'Login'}/>
                </Link>
            </form>
        </div>
    );
}

export default Login;