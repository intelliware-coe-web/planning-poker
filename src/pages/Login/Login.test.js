import React from 'react';
import Login from './Login';

it('login should match snapshot', () => {
    expect(Login()).toMatchSnapshot();
});