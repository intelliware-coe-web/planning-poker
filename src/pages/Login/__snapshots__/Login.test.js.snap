// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`login should match snapshot 1`] = `
<div
  className="login"
>
  <p>
    Welcome to Planning Poker
  </p>
  <img
    alt="logo"
    className="login-logo"
    src="logo.svg"
  />
  <form>
    <EmailInput />
    <PasswordInput />
    <Link
      to="/rooms"
    >
      <Button
        name="Login"
      />
    </Link>
  </form>
</div>
`;
